delivery_schenker/models/delivery_carrier.py

```diff
@@ -165,6 +166,18 @@ class DeliveryCarrier(models.Model):
         help="If not delivery package or the package doesn't have defined the packaging"
         "it will default to this type",
     )
+    schenker_address_number = fields.Char(
+        "Address ID",
+        help="ID assigned by Schenker to you.\nWill be part of the sender or "
+        "if set, the invoice address.\nIf you don't want to send it, use 0",
```

Comment:
According to the help:

```suggestion
        "if set, the invoice address.\nIf you don't want to send it, use 0",
        default=0,
```